src/runtime/hexagon_host.cpp

```diff
@@ -71,6 +73,19 @@ WEAK void poll_log(void *user_context) {
     }
 }
 
+WEAK int get_remote_profiler_func() {
+    if (!remote_poll_profiler_func) {
+        // This should only have been called if there's a remote profiler func installed.
+        error(NULL) << "Hexagon: remote_poll_profiler_func not found\n";
+    }
+
+    int func = 0;
+    if (int result = remote_poll_profiler_func(&func)) {
+        error(NULL) << "Hexagon: remote_poll_profiler_func failed " << result << "\n";
```

Comment:
Can you change this to debug? I think that if this happens, it might obscure more useful error reports when the pipeline itself returns.